pages/vi/profiles/cindypsu.md

```diff
@@ -0,0 +1,22 @@
+# Cynthia Hilgeman _(Cindy)_
+---
+Location: Bozeman, Montana (U.S.A)   
+Time Zone: Mountain Standard Time (MST), UTC/GMT -7hours  
+MS Windows 10 Pro, 10.0.16299
+
+This is me:
+
+![Alt text](https://avatars0.githubusercontent.com/u/35432613?s=400&u=a0637d3f2d86623519841fb953df4592bdca9f80&v=4 "Me")
```

Comment:
Curious to know if you got this image embedding syntax from some other PRs? Because this is the 3rd time I see an image embedding with a `title` but leaving `alt` to be placeholder "alt text"...

Reply:
Honestly, I believe I pulled it from the example provided in the "First Steps" section for Step 3. I was playing around with the markdown syntax and I tried a couple different variations, but this is the version that I left in my profile. However, I am happy to make the fix. I will give it a go and post an update here once I've completed it.

Reply:
@xyb994 Fix has been applied to my profile.

Reply:
@CindyPSU Sorry if the link misguided you, it looks like you used another way to embed the image but did not replace `alt text` to something meaningful or leave it blank depending on the situation.